examples/rhoai-quickstart/feast-demo-quickstart.ipynb

```diff
@@ -0,0 +1,1530 @@
+    "!rm -rf my_feast_project\n",
+    "!feast init my_feast_project"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3beb2f85-3e28-4afa-8227-30271dd00c7f",
+   "metadata": {},
+   "source": [
+    "Above output displays where the feast repo has been created. It may differ based on the environment configuration. On the example currently it is placing at `/opt/app-root/src/feast/examples/rhoai-quickstart/my_feast_project`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "308610d4-9174-4972-a785-237c0b0477d8",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/opt/app-root/src/feast/examples/rhoai-quickstart/my_feast_project/feature_repo\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/opt/app-root/lib64/python3.9/site-packages/IPython/core/magics/osm.py:417: UserWarning: using dhist requires you to install the `pickleshare` library.\n",
+      "  self.shell.db['dhist'] = compress_dhist(dhist)[-100:]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Going to change the current directory to feature_repo so that we can execute feast CLI commands.\n",
+    "%cd my_feast_project/feature_repo"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7f5212d8-cd46-4017-a762-91a4fb56e0d5",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 36K\n",
+      "drwxr-xr-x. 4 1001130000 1001130000 4.0K Sep 24 03:07 .\n",
+      "drwxr-xr-x. 3 1001130000 1001130000 4.0K Sep 24 03:07 ..\n",
+      "drwxr-xr-x. 2 1001130000 root       4.0K Sep 24 03:07 data\n",
+      "-rw-r--r--. 1 1001130000 1001130000 5.2K Sep 24 03:07 example_repo.py\n",
+      "-rw-r--r--. 1 1001130000 1001130000  372 Sep 24 03:07 feature_store.yaml\n",
+      "-rw-r--r--. 1 1001130000 1001130000    0 Sep 23 16:09 __init__.py\n",
+      "drwxr-xr-x. 2 1001130000 1001130000 4.0K Sep 23 16:09 __pycache__\n",
+      "-rw-r--r--. 1 1001130000 1001130000 4.3K Sep 23 16:09 test_workflow.py\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Inspect the feast repo path files. Going to describe each file/folder purpose.\n",
+    "!ls -lah"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2fa8d4c3-8f7d-4758-8048-945ebd0af53f",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Now the feast repo has been created for you. Inspect the below files.\n",
+    "\n",
+    "`data` contains the parquet file data used to demonstrate this example.\n",
+    "\n",
+    "`example_repo.py` file will have the code to create feast objects such as FeatureView, FeatureServices and OnDemandFeatureViews required to demonstrate this example.\n",
+    "[my_feast_project/feature_repo/example_repo.py](./my_feast_project/feature_repo/example_repo.py)\n",
```

Comment:
links are broken as we're not storing the project files in git. not sure about the best option to show the path

Reply:
Links will work after executing the `feast init` command. I want the user to review the files explicitly so provided the link with detailed description. Also added more description to address some of your other comments as well. Please let me know if it is still not very clear.